config/terraform/application/config/migration.tfvars.json

```diff
@@ -5,5 +5,6 @@
     "enable_logit": true,
     "worker_replicas": 6,
     "worker_memory_max": "2Gi",
-    "postgres_flexible_server_sku": "GP_Standard_D4ds_v5"
+    "postgres_flexible_server_sku": "GP_Standard_D4ds_v5",
+    "rails_env": "migration"
```

Comment:
It might be better to add RAILS_ENV to the migration.yml in the same directory. That would then get loaded via https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/config/terraform/application/application.tf#L19 and you wouldn't need any tf variable.